### PR TITLE
Feature - Building Cat App

### PR DESCRIPTION
**Pull request by tradm**

### CODE REVIEW

File: src/components/searchResult/resultCard/ResultCard.module.scss [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    height: 200px;
+  }
+}
+
+.resultCardImage {
+  border-bottom-left-radius: 8px;
+  border-top-left-radius: 8px;
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
```
Comment (pdziuba) [inline src/components/searchResult/resultCard/ResultCard.module.scss:32]: first time seeing this, what does it do?

File: src/components/searchResult/resultCard/ResultCard.module.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  @media screen and (max-width: 978px) {
```
Comment (pdziuba) [inline src/components/searchResult/resultCard/ResultCard.module.scss:34]: this is cool :) just wanted to hear your opinion on mobile first approach, have you tried that?

File: src/repositories/app-repository.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      mode: 'cors',
+    });
+
+    await handleError(getCatTagsResponse);
+    return getCatTagsResponse.json();
+  },
+
+  getCats: async (query: string) => {
+    const getCatsResponse = await fetch(
+      `${url}/cats${query ? `${query}` : ''}`,
```
Comment (pdziuba) [inline src/repositories/app-repository.ts:27]: what if query contains spaces? shouldn't we encode/escape the query string somehow?

File: src/components/searchResult/resultCard/ResultCard.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        </Stack>
+
+        <Stack className={styles.resultCardDetailStack}>
+          <CalendarTodayOutlinedIcon />
+          <Typography>{date}</Typography>
+        </Stack>
+
+        <Stack className={styles.resultCardPills}>
+          {cat.tags.map((tag, ind) => (
+            <CustomPill key={ind} label={tag} onClick={handlePillClick} />
```
Comment (pdziuba) [inline src/components/searchResult/resultCard/ResultCard.tsx:49]: I've heard that using index for key in the list is not the best idea :)

**Review by pdziuba — COMMENTED** [review]
Overall I really like what you have done. I have couple comments but overall very good job!

**Comment by benediktmartinez** [comment]
few comments:

- feeling lucky button never gives results. how would you implement it?
- when selecting multiple tags in the search field, it seems none of them are passed to the API
- selecting a pill on the results page doesn't trigger new search. where is the problem?

**Pull request closed without merging**